app/views/storage.html

```diff
@@ -55,7 +55,9 @@
                 </tbody>
                 <tbody ng-if="(pvcs | hashSize) > 0">
                   <tr ng-repeat="pvc in pvcs | orderObjectsByDate : true">
-                    <td data-title="Name"><a ng-href="{{pvc | navigateResourceURL}}">{{pvc.metadata.name}}</a></td>
+                    <td data-title="Name"><a ng-href="{{pvc | navigateResourceURL}}">{{pvc.metadata.name}}</a>
+                      <span ng-if="pvc | annotation : 'volume.beta.kubernetes.io/storage-class'" class="text-muted"> using storage class {{pvc | annotation : 'volume.beta.kubernetes.io/storage-class'}}</span>
```

Comment:
I agree, this will make things much nicer and be easier to update in the futures.  Changes added.